frontend/src/components/Map/Map.tsx

```
@@ -0,0 +1,102 @@
+
+const melbourneGeoData: [number, number] = [-37.8136, 144.9631];
+
+const MapZoomHandler = () => {
+  const map = useMap();
+  useEffect(() => {
+    map.scrollWheelZoom.enable();
+  }, [map]);
+  return null;
+};
+
+const Map = () => {
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)